Developers/Git_Basics.rst

```diff
@@ -19,6 +19,13 @@ this will do:
 
     $ sudo apt-get install git-all
```

Comment:
commit message: in body use Adds, not Add.
also, correct is "an example ON how"